sources/packages/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -155,7 +169,7 @@ export class AssessmentControllerService {
    */
   private populateDisbursementAwardValues(
     disbursementSchedules: DisbursementSchedule[],
-    includeDocumentNumber = false,
+    includeDocumentNumber = true,
```

Comment:
Is it not safe to keep it false as it was? If the parameter is not provided the value will not be included by mistake.